Lib/pickletools.py

```diff
@@ -2251,7 +2251,7 @@ def genops(pickle):
 ##############################################################################
 # A pickle optimizer.
 
-def optimize(p):
+def optimize(pickled):
```

Comment:
Why not `pickle`? In any case I would left renaming the argument to other issue. There are many related changes should be made (updating the documentation, docstrings).

Reply:
Because it's the name of the module and I saw "pickled" used elsewhere.

I changed it because I had trouble understanding the meaning of that single letter variable when reading the code.

Reply:
Actually, the "pickled" notation was used in another project. I reverted to old notation for now.